docs/specs/legacy.yml

```diff
@@ -70,6 +70,7 @@ outputs:
         <meta name=\"depot_name\" content=\".\" />
         <meta name=\"document_id\" content=\"fdb5a0d7-b68e-c465-9c5e-54a5cda94902\" />
         <meta name=\"document_version_independent_id\" content=\"92a5a770-f05f-b8bb-81c9-a8d786a9e608\" />
+        <meta content=\"charlie\" name=\"git_contributor_information\">
```

Comment:
Add it to `OpsMetadata.json`. `htmlMetaHidden` section, this list does not need to appear in the final HTML.

Reply:
Will create a new pr to do this.